src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+HTML,
+body,
+header,
+ul,
+li {
   margin: 0;
+  padding: 0;
+}
+
+.nav {
+  height: 100%;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  /* height: 60px; */
+}
+
+.button {
+  margin-left: 20px;
+  /* height: 100%; */
+  height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.button:first-child {
+  margin-left: 0;
+}
+
+.link {
+  height: 100%;
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
```

Comment:
Better will be to set this as default font-family for the page(in html selector)